pytest-selenium/pages/base.py

```diff
@@ -44,3 +44,23 @@ def offscreen_click(self, element):
         # https://stackoverflow.com/a/39918249
         element.send_keys(Keys.ENTER)
         return element
+
+    def click_and_check_reference_object_has_changed(self, click_element, ref_object, ref_property):
```

Comment:
checking to make sure the page title or chapter title have been changed are nifty but not completely accurate ways to make sure the page has loaded, i liked the previous implementation of this method because it abstracted the exact method of telling when the page was loaded, and if we implement a better way in the future we'd only have to do it in one place.

Reply:
i don't see how it abstracted anything away as it was hardcoded to always use the title. It was also checking the title on the parent class which didn't have a title which was also confusing to me. in this case here, any element that is clicked that also causes a side effect can be checked. 

I'm in full agreement that there should be something better for detecting when a page is fully loaded. I wanted to suggest initially that maybe on these navigation transitions maybe the `rex-loaded` attribute should be removed and added again when the page is loaded. @TomWoodward

Reply:
yea i agree that a better solution is possible and we should do it, but it doesn't have to happen right now and we can just change it in this method when we get around to it

Reply:
i agree. i've reverted my changes.